sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Infrastructure/EnvironmentVariables.cs

```diff
@@ -3,18 +3,80 @@
-
-        // Event Hub Names
-        public const string EventHubBufferedProducerTest = "EVENTHUB_NAME_BUFFERED_PRODUCER_TEST";
-        public const string EventHubEventProducerTest = "EVENTHUB_NAME_EVENT_PRODUCER_TEST";
-        public const string EventHubBurstBufferedProducerTest = "EVENTHUB_NAME_BURST_BUFFERED_PRODUCER_TEST";
-        public const string EventHubConcurrentBufferedProducerTest = "EVENTHUB_NAME_CONCURRENT_BUFFERED_PRODUCER_TEST";
-    }
+    // Shared Resources
+
+    /// <summary>
+    ///   The name of the environment variable that holds the instrumentation key for the
+    ///   Application Insights resource for the test runs.
+    /// </summary>
+    ///
+    public const string ApplicationInsightsKey = "APPINSIGHTS_INSTRUMENTATIONKEY";
+
+    /// <summary>
+    ///   The name of the environment variable that holds the connection string for the
+    ///   Event Hubs Namespace resource for the test runs.
+    /// </summary>
+    ///
+    public const string EventHubsConnectionString = "EVENTHUB_NAMESPACE_CONNECTION_STRING";
+
+    // Event Hub Names
+
+    /// <summary>
+    ///   The name of the environment variable that holds the name of the Event Hub resource
+    ///   for the Event Hub Buffered Producer test scenario.
+    /// </summary>
+    ///
+    public const string EventHubBufferedProducerTest = "EVENTHUB_NAME_BUFFERED_PRODUCER_TEST";
+
+    /// <summary>
+    ///   The name of the environment variable that holds the name of the Event Hub resource
+    ///   for the Event Hub Producer test scenario.
+    /// </summary>
+    ///
+    public const string EventHubEventProducerTest = "EVENTHUB_NAME_EVENT_PRODUCER_TEST";
+
+    /// <summary>
+    ///   The name of the environment variable that holds the name of the Event Hub resource for the
+    ///   Event Hub burst Buffered Producer test scenario.
+    /// </summary>
+    ///
+    public const string EventHubBurstBufferedProducerTest = "EVENTHUB_NAME_BURST_BUFFERED_PRODUCER_TEST";
+
+    /// <summary>
+    ///   The name of the environment variable that holds the name of the Event Hub resource for the processor test scenario.
+    /// </summary>
+    ///
+    public const string EventHubProcessorTest = "EVENTHUB_NAME_PROCESSOR_TEST";
+
+    /// <summary>
+    ///   The name of the environment variable that holds the name of the storage blob resource for the processor test
+    ///   scenario.
+    /// </summary>
+    ///
+    public const string StorageBlobProcessorTest = "STORAGE_BLOB_PROCESSOR_TEST";
+
+    /// <summary>
+    ///   The name of the environment variable that holds the name of the storage account connection string for the processor test
+    ///   scenario.
+    /// </summary>
+    ///
+    public const string StorageAccountProcessorTest = "STORAGE_ACCOUNT_PROCESSOR_TEST";
+
+    // Job Index Information
+
+    /// <summary>
+    ///   The name of the environment variable that holds the index of the kubernetes pod. This variable should
```

Comment:
```suggestion
    ///   The name of the environment variable that holds the index of the Kubernetes pod. This variable should
```
nit:  Just making it consistent with the other uses in this paragraph.